lib/substitution_map_extractor.ex

```diff
@@ -48,9 +65,11 @@ defmodule SubstitutionMapExtractor do
     |> Enum.map(&to_order_number_name_map/1)
   end
 
-  def convert_mapped(order_number_name_map) do
+  def convert_mapped(order_number_name_map, previous_file_last_order_number) do
```

Comment:
Note that we need to put the previous file's last order number in a bunch of signatures. Messy. A candidate for some simplification with a `struct` perhaps? Not so important for a script like this.